python/testing/TestPythonCamera.py

```diff
@@ -38,5 +38,5 @@
 camera.pitch(angle)
 
 camera.setCurrentAsDefault()
-camera.resetToBounds()
+camera.resetToBounds(0.9)
```

Comment:
can we have default values in pybind11 ?

Reply:
`.def("resetToBounds", &f3d::camera::resetToBounds, py::arg("zoomFactor") = 0.9);` should work